### ISSUE TITLE
[NBug] Object reference not set to an instance of an object.

### ISSUE THREAD

**Issue by LaurensvandenBerg** [gitextensions/gitextensions#7193]
## Current behaviour

Pressing the push button tries to make a github plugin connection, even though the repository is not on github.

## Expected behaviour
not calling github on non github repo's.

## Steps to reproduce
press push on a non-github repository.

## Error Details
```
System.NullReferenceException: Object reference not set to an instance of an object.
   at GitHub3.GitHub3Plugin.GetHostedRemotesForModule()
   at GitHub3.GitHub3Plugin.GitModuleIsRelevantToMe()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
   at GitUI.CommandsDialogs.FormPush.FormPushLoad(Object sender, EventArgs e)
   at System.EventHandler.Invoke(Object sender, EventArgs e)
   at System.Windows.Forms.Form.OnLoad(EventArgs e)
   at GitUI.GitExtensionsForm.OnLoad(EventArgs e)
   at System.Windows.Forms.Form.OnCreateControl()
   at System.Windows.Forms.Control.CreateControl(Boolean fIgnoreVisible)
   at System.Windows.Forms.Control.CreateControl()
   at System.Windows.Forms.Control.WmShowWindow(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.Form.WmShowWindow(Message& m)
   at ResourceManager.GitExtensionsFormBase.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

## Environment
- Git Extensions 3.2.1.6628
- Build bc07663c3c893f1d896520e0bdcfbe35cfe4b729
- Git 2.23.0.windows.1
- Microsoft Windows NT 10.0.18362.0
- .NET Framework 4.8.3815.0
- DPI 96dpi (no scaling)

**Comment by pmiossec** [comment]
>Pressing the push button tries to make a github plugin connection, even though the repository is not on github.

No, it is not.

The `GitModuleIsRelevantToMe()` method of the `GitHub3Plugin` plugin is called to know if you have a remote that is a GitHub one to know if the checkbox "Create a pull request after push" must be enabled.

![image](https://user-images.githubusercontent.com/460196/65421575-55122a00-de04-11e9-8bc5-c252b6c87dbc.png)

There are no called to GitHub made.

But that doesn't explain why you have this exception...

**Comment by RussKie** [comment]
Can you please provide steps to reproduce? Some screenshots can be useful too.
In all the years of using GE with various hosters (including private behind proxies and firewalls) I never had that exception

**Comment by LaurensvandenBerg** [comment]
After trying everything in all kinds of repositories, i am unable to repoduce the issue myself...

**Comment by gerhardol** [comment]
This is still something that should be handled, more info is welcome.
The functionality to "Create pull request after push" is new or at least changed in 3.2

**Comment by pmiossec** [comment]
Closed with #7206
